src/Map/ReactMap.tsx

```diff
@@ -31,33 +27,32 @@ export function ReactMap(props: IMap) {
   // eslint-disable-next-line react-hooks/exhaustive-deps -- No need for selectedBuilding in the deps list
   }, [buildingsToMap]);
 
-  return (
-    <LoadScript
-      googleMapsApiKey={ firebaseConfig.apiKey }
-      libraries={ LIBRARIES }
-      language="en"
-      version="quarterly"
+  const { isLoaded } = useJsApiLoader({
+    id: 'google-map-script',
+    googleMapsApiKey: firebaseConfig.apiKey,
+
+  })
+
+  return isLoaded ? (
```

Comment:
Might be better to use an if statement instead of a ternary.

```suggestion
  if (!isLoaded) {
    return null
  }

  return (
```